python/GafferSceneUI/CryptomatteUI.py

```diff
@@ -282,6 +282,18 @@ def __layerPresetValues( plug ) :
 
 		],
 
+		"manifestScene" : [
+
+			"description",
+			"""
+			A scene containing locations representing the contents of the Cryptomatte manifest.
+			""",
+
+			"plugValueWidget:type", None,
+			"nodeGadget:type", None,
```

Comment:
I think this would need to be "nodule:type", but since the nodule isn't showing up, I presume it's not actually necessary.

Reply:
D'oh!